index.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
+
```

Comment:
Both or just this one? I already get criticized for not adding enough blank lines. :stuck_out_tongue: